examples/nextjs-with-typescript/src/pages/_app.tsx

```diff
@@ -4,8 +4,8 @@ import { AppProps } from 'next/app';
 import { ThemeProvider } from '@mui/material/styles';
 import CssBaseline from '@mui/material/CssBaseline';
 import { CacheProvider, EmotionCache } from '@emotion/react';
-import theme from '../src/theme';
-import createEmotionCache from '../src/createEmotionCache';
+import theme from '../../styles/theme';
+import createEmotionCache from '../lib/createEmotionCache';
```

Comment:
```suggestion
import createEmotionCache from '../utils/createEmotionCache';
```

Let's use `utils`, as we do in the rest of the codebase.

Reply:
Cool, I'll change that.